benchmark/child_process/spawn-echo.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 var common = require('../common.js');
 var bench = common.createBenchmark(main, {
-  thousands: [1]
+  len: [1000]
```

Comment:
I'm not sure about this change. In the other benchmarks, `len` is a message length in bytes I think. Here, it is a number of iterations, so `len` is kind of misleading because it's not a length. We tend to use `n` for this sort of thing instead.

I'm not opposed to this, but would be curious what others thought. @nodejs/benchmarking @mscdex